metricflow/engine/metricflow_engine.py

```diff
@@ -443,9 +444,13 @@ def simple_dimensions_for_metrics(self, metric_names: List[str]) -> List[Dimensi
             Dimension(name=dim.qualified_name)
             for dim in self._semantic_model.metric_semantics.element_specs_for_metrics(
                 metric_specs=[MetricSpec(element_name=mname) for mname in metric_names],
-                dimensions_only=True,
-                exclude_derived_time_granularities=True,
-                exclude_local_linked_primary_time=True,
+                without_any_property=frozenset(
```

Comment:
For my understanding here - are identifiers and dimensions the only things that might be returned from `element_specs_for_metrics`? So specifying without property `IDENTIFIER` is equivalent to specifying `dimensions_only`?

Reply:
That is correct.